homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -217,7 +217,7 @@ def _async_update_current_cover_position(self):
         _LOGGER.debug("Raw data update: %s", self._shade.raw_data)
         position_data = self._shade.raw_data.get(ATTR_POSITION_DATA, {})
         if ATTR_POSITION1 in position_data:
-            self._current_cover_position = position_data[ATTR_POSITION1]
+            self._current_cover_position = int(position_data[ATTR_POSITION1])
```

Comment:
What value is in `position_data[ATTR_POSITION1]` when its being cast to an `int`?